File: app/src/main/java/com/gago/david/myland/Models/TaskTypeObject.java
package com.gago.david.myland.models;

import java.io.Serializable;

public class TaskTypeObject implements Serializable {
    public String name;
    public String description;

    public TaskTypeObject(String name, String description) {
        this.name = name;
        this.description = description;
    }

    @Override
    public String toString() {
        return "TaskTypeObject{" +
                "name='" + name + '\'' +
                ", description='" + description + '\'' +
                '}';
    }
}

File: app/src/main/java/com/gago/david/myland/models/TaskTypeObject.kt
package com.gago.david.myland.models

import java.io.Serializable

class TaskTypeObject(var name: String, var description: String) : Serializable {
    override fun toString(): String {
        return "TaskTypeObject{" +
                "name='" + name + '\'' +
                ", description='" + description + '\'' +
                '}'
    }
}